src/core/engine/processors/mongo_order_status_history_distribution.py

```diff
@@ -0,0 +1,59 @@
+from abc import ABCMeta
```

Comment:
I think this is an ETL instead of a raw processor

Reply:
Done.